### PR TITLE
Include fonts as assets in releases

### LINKED ISSUE #176: Add font files to `Assets` when release
We could add our font files to `Assets` when we release them.

See the `Assets` section at https://github.com/simple-icons/simple-icons-font/releases.

Just like Fira Code does https://github.com/tonsky/FiraCode/releases.

### PR DESCRIPTION
**Pull request by mondeja**
Resolves #176

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -73,3 +89,8 @@ jobs:
           release_name: Release ${{ steps.get-version.outputs.version }}
           body: |
             See https://github.com/simple-icons/simple-icons/releases/tag/${{ steps.get-version.outputs.version }}
+      - uses: shogo82148/actions-upload-release-asset@v1
+        name: Upload release assets
+        with:
+          upload_url: ${{ steps.create-release.outputs.upload_url }}
+          asset_path: font/*
```
Comment (LitoMore) [inline .github/workflows/publish.yml:96]: We could pack all files into a zip file. It would be better if it could include LICENSE and DISCLAIMER.

Comment (mondeja) [reply .github/workflows/publish.yml]: Done. Also enabled dry-run publish workflow runs. You can see the content of the released ZIP at "Publish" workflow summaries: https://github.com/simple-icons/simple-icons-font/actions/runs/3141963909

**Comment by LitoMore** [comment]
CleanShot 2022-09-28 at 16 35 50@2x

The DISCLAIMER is in the node_modules directory. We could put it to same directory as LICENSE.

We could make its structure to:

CleanShot 2022-09-28 at 16 42 11@2x

**Comment by mondeja** [comment]
> We could put it to same directory as LICENSE.

Thanks, changed to this 👍🏼

**Review by LitoMore — APPROVED** [review]

**Pull request merged**